ts/Series/Sankey/SankeySeries.ts

```diff
@@ -537,6 +537,24 @@ class SankeySeries extends ColumnSeries {
     public createNodeColumns(): Array<SankeySeries.ColumnArray> {
         const columns: Array<SankeySeries.ColumnArray> = [];
 
+        // Get columns offset including all sibiling and cousins etc.
+        function getOffset(node: SankeyPoint): number {
```

Comment:
I wish I could move this code from SankeySeries to OrganizationSeries. Correct me if I'm wrong, but the whole `layout === "hanging"` part below belongs to OrgSeries, right?

Reply:
Yes, I can't see a scenario for hanging layout in Sankey.

Reply:
Thanks! Will do it